install_config/persistent_storage/persistent_storage_azure.adoc

```diff
@@ -59,8 +59,23 @@ xref:../../install_config/configuring_azure.adoc#install-config-configuring-azur
 for Azure Disk], all that is required for {product-title} and Azure is an Azure
 Disk Name and Disk URI and the `PersistentVolume` API.
 
-[[azure-creating-persistent-volume]]
+[[azure-disk-regional-cloud]]
+== Configuring Azure Disk for regional cloud
+
+Azure has multiple regions on which to deploy an instance. To specify a desired region, add the following to the *_azure.conf_* file:
```

Comment:
s/Azure/Microsoft Azure 
I noticed this in our Glossary yesterday. Up to you if you want to modify other instances as well.